config/sanctum.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
Can probably delete this one as we don't make any customisations to it for Mailcoach